src/pynwb/io/__init__.py

```diff
@@ -1,3 +1,5 @@
+# flake8: noqa: F401
```

Comment:
This flake8 exception should be handled already by https://github.com/NeurodataWithoutBorders/pynwb/blob/ab493464ddc65df4724b5d82354e31fadc91880d/setup.cfg#L26

Reply:
I removed it.